test/index.js

```diff
@@ -25,23 +29,30 @@ fixtures.valid.rsa.forEach(function (f) {
     priv = new Buffer(f.private, 'base64')
   }
 
-  test(f.message, function (t) {
-    var bSign = bCrypto.createSign(f.scheme)
-    var nSign = nCrypto.createSign(f.scheme)
-    var bSig = bSign.update(message).sign(priv)
-    var nSig = nSign.update(message).sign(priv)
+  var schemes = [f.scheme]
+  if (fixtures.aliases[f.scheme]) {
+    schemes.push(fixtures.aliases[f.scheme])
+  }
+  schemes.forEach(function (scheme) {
+    test(f.message + ' (' + scheme + ')', function (t) {
+      var bSign = bCrypto.createSign(scheme)
+      var nSign = nCrypto.createSign(scheme)
+      var bSig = bSign.update(message).sign(priv)
+      var nSig = nSign.update(message).sign(priv)
 
-    t.equals(bSig.length, nSig.length, 'correct length')
-    t.equals(bSig.toString('hex'), nSig.toString('hex'), 'equal sigs')
-    t.equals(bSig.toString('hex'), f.signature, 'compare to known')
+      t.equals(bSig.length, nSig.length, 'correct length')
+      t.equals(bSig.toString('hex'), nSig.toString('hex'), 'equal sigs')
 
-    t.ok(nCrypto.createVerify(f.scheme).update(message).verify(pub, nSig), 'node validate node sig')
-    t.ok(nCrypto.createVerify(f.scheme).update(message).verify(pub, bSig), 'node validate browser sig')
+      t.equals(nSig.toString('hex'), f.signature, 'node compare to known')
```

Comment:
Tests were failing before I added the id/tag to the other definitions, so I added this as a sanity check. This was consistently "OK", meaning node was doing the expected thing.

Reply:
ok this is a good check